app/src/main/kotlin/com/wire/android/ui/common/WireDialog.kt

```diff
@@ -171,6 +182,44 @@ private fun WireDialogPreview() {
     }
 }
 
+@Preview(showBackground = true)
+@Composable
+private fun WireDialogPreview2() {
```

Comment:
WireDialogPreview2() does not tell me alot, maybe a better name here ?

Reply:
Done!